StokesProblem.hh

```diff
@@ -0,0 +1,231 @@
+#ifndef _STOKESPROBLEM_HH_
+#define _STOKESPROBLEM_HH_
+
+#include <vector>
+
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function.h>
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/synchronous_iterator.h>
+#include <deal.II/base/work_stream.h>
+
+#include <deal.II/lac/block_vector.h>
+#include <deal.II/lac/full_matrix.h>
+
+#include <deal.II/lac/block_sparse_matrix.h>
+#include <deal.II/lac/solver_cg.h>
+#include <deal.II/lac/precondition.h>
+#include <deal.II/lac/constraint_matrix.h>
+#include <deal.II/lac/precondition_block.h>
+
+#include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/sparse_matrix.h>
+#include <deal.II/lac/dynamic_sparsity_pattern.h>
+#include <deal.II/lac/compressed_sparsity_pattern.h>
+#include <deal.II/lac/sparse_ilu.h>
```

Comment:
You don't need ```sparse_ilu.h``` There are way too many ```#include``` here. Some can be deleted and others can be moved in the .cc file.